finagle-http/src/main/scala/com/twitter/finagle/Http.scala

```diff
@@ -268,6 +268,13 @@ object Http extends Client[Request, Response] with HttpRichClient
     def withCompressionLevel(level: Int): Client =
       configured(param.CompressionLevel(level))
 
+
+    /**
+     * Enable the collection of HTTP specific metrics. See [[http.filter.StatsFilter]].
+     */
+    def enableHttpStats(): Client =
```

Comment:
There is `noFailFast` on `Client/Server` (it's a client builder thing). We use `with`-prefixed methods here instead. And it's totally fine to have `with`-API w/o an argument (we do that all the time). See [Design Principles for the `with`-API](http://twitter.github.io/finagle/guide/Configuration.html#design-principles) (item number 2).